### PR TITLE
Html

### PR DESCRIPTION
**Pull request by Ajay-Dhangar**

### CODE REVIEW

File: docs/html-mastery/basic-level/intro-html.md [hunk 1]
```diff
@@ -1,136 +0,0 @@
...
-
-5. **How do you set the title of a webpage?**
-   - The title of a webpage is set using the `<title>` element within the `<head>` section of the HTML document. The text placed within the `<title>` tags will be displayed as the title of the webpage in the browser's title bar or tab.
-
-6. **How do you view an HTML file in a web browser?**
-   - To view an HTML file in a web browser, you need to save the HTML file with the .html extension and open it in a web browser by using the "Open File" option or dragging the file into the browser window. The web browser will render and display the HTML content.
-
-7. **Can you explain the difference between the `<head>` and `<body>` sections in an HTML document?**
-   - The `<head>` section contains metadata about the document, such as the title, linked CSS or JavaScript files, and other related information. It is not visible on the webpage. The `<body>` section contains the visible content of the webpage, including text, images, links, and other elements that users can see and interact with.
-
```
Comment (Ajay-Dhangar) [inline docs/html-mastery/basic-level/intro-html.md:136]: Don't delete

**Review by Ajay-Dhangar — COMMENTED** [review]
can you do to resolve issues

File: docs/html-mastery/basic-level/intro-html.md (hunk 1 above)
Comment (ShivayaCoder) [reply docs/html-mastery/basic-level/intro-html.md:136]: yah that's show some issues

**Review by ShivayaCoder — CHANGES_REQUESTED** [review]
Resolve issues

**Pull request closed without merging**